vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddExternalLinkController.java

```diff
@@ -53,10 +57,9 @@
             filename = file.getOriginalFilename();
         }
         DisplayType type = displayType.isEmpty() ? null : DisplayType.valueOf(displayType);
-        ExternalLinkDisplayMode externalLinkOpenMode = howToOpen.isEmpty() ? null
-                : ExternalLinkDisplayMode.valueOf(howToOpen);
-        IExternalLinkDisplay display = externalLinkManager.createLink(title, id, new Float(x), new Float(y), 0,
-                externalLink, title, type, linkImage, filename, externalLinkOpenMode);
+
+        IExternalLinkDisplay display = externalLinkManager.createLink(title, id, new Float(x), new Float(y), 0, externalLink, title, desc, type, linkImage, filename, imageId);
```

Comment:
It looks like `externalLinkOpenMode` was dropped here.

Reply:
@jdamerow for the new changes externalLinkOpenMode is not being used only